graphql/resolvers.go

```diff
@@ -192,11 +192,33 @@ func (r *volumeResolver) Host(ctx context.Context, obj *restModel.APIVolume) (*r
 func (r *queryResolver) HasVersion(ctx context.Context, id string) (bool, error) {
 	// We do not check the error here because we only want to return an error if the id is not
 	// a valid patch or version id.
-	if v, _ := r.sc.FindVersionById(id); v != nil {
+	v, err := r.sc.FindVersionById(id)
+	if err != nil {
+		apiErr, ok := err.(gimlet.ErrorResponse)
+		if !ok {
+			return false, InternalServerError.Send(ctx, fmt.Sprintf("Error finding version %s: %s", id, err.Error()))
+		}
+		if apiErr.StatusCode != http.StatusNotFound {
+			return false, InternalServerError.Send(ctx, fmt.Sprintf("Error finding version %s: %s", id, err.Error()))
```

Comment:
can use apiErr.Message here now instead of err.Error() (this just removes the status code)